<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right: 2px solid black;</string>
    </property>
    <widget class="QGroupBox" name="patientDetails">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>361</width>
       <height>621</height>
      </rect>
     </property>
     <property name="title">
      <string>Patient Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="imageSection" native="true">
        <property name="styleSheet">
         <string notr="true">border-right:none;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="textDetails">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="medHistory">
          <property name="styleSheet">
           <string notr="true">margin: 20px</string>
          </property>
          <property name="text">
           <string>View Medical History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newPriscrip">
          <property name="styleSheet">
           <string notr="true">margin: 20px</string>
          </property>
          <property name="text">
           <string>Create New Prescription</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="searchInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchPatient">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Patient by Email ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>1511</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="medicalHistory">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>1231</width>
        <height>961</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Medical History</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="newPriscription">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>1231</width>
        <height>961</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>New Priscription</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionPriscription"/>
   </widget>
   <widget class="QMenu" name="menuGet">
    <property name="title">
     <string>Get</string>
    </property>
    <addaction name="actionPatient_Details"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuNew"/>
   <addaction name="menuGet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPriscription">
   <property name="text">
    <string>Priscription</string>
   </property>
  </action>
  <action name="actionPatient_Details">
   <property name="text">
    <string>Patient Details</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
